addon/components/flash-message.js

```diff
@@ -59,6 +60,14 @@ export default Component.extend({
     }
   }),
 
+  // Observer used here for backwards compatibility with Ember 1.11/1.12, replace
+  // with on('didRender') once possible.
+  _startTimers: on('didInsertElement', observer('flash', function() {
```

Comment:
That's odd. I'll investigate and let you know, I'm very against using observers unless strictly necessary